unicodetools/org/unicode/unittest/TestIdnaTest.java

```diff
@@ -154,16 +154,15 @@ private void checkAscii(TestLine tl, IdnaChoice idnaChoice, String detailedSourc
             f = " toAsciiN(";
             break;
         }
-        String f = idnaChoice == IdnaChoice.transitional ? " toAsciiT(" : " toAsciiN(";
```

Comment:
This was broken when restored from SVN. Needs more work; this is just to get it to compile...